tools/ui-components/src/global-element-styles.css

```diff
@@ -1,5 +1,5 @@
 /*! 
-  element styles of the minified boostrap 3 css file 
+  element styles of the minified bootstrap 3 css file 
```

Comment:
This should actually be capitalized because it's referring to the framework "Bootstrap 3".

Reply:
Fixed in https://github.com/freeCodeCamp/freeCodeCamp/pull/51246/commits/4d6f6ac2c79cb7cf674a5f911c204c1deef099a3